### ISSUE TITLE
Connect to external databases

### ISSUE THREAD

**Issue by syphax-bouazzouni** [agroportal/project-management#445]
See the way Finto connects to a database:

https://finto.fi/koko/en/page/p35196

with the image ...

![Image](https://github.com/agroportal/project-management/assets/29259906/7b6a00bf-56c5-4e30-9316-ba6023bdd0a4)

It seems to be connected to Wikipedia and this [https://finna.fi/Search/Results?bool0%5B0%5D=OR&lookfor0%5B0%5D=topic_facet%3Alay[…]lds&join=AND&filter%5B0%5D=%7Eformat%3A0%2FImage%2F&callback=%3F](https://finna.fi/Search/Results?bool0%5B0%5D=OR&lookfor0%5B0%5D=topic_facet%3Alayout%5E2&lng=en-gb&limit=&type=AllFields&join=AND&filter%5B0%5D=%7Eformat%3A0%2FImage%2F&callback=%3F)

![Image](https://github.com/agroportal/project-management/assets/29259906/77fc3091-732f-473e-8f21-513da5ddd72d)

**Comment by jonquet** [comment]
Related to https://github.com/agroportal/project-management/issues/285
Related to https://github.com/agroportal/project-management/issues/4

**Comment by syphax-bouazzouni** [comment]
See also the example https://vocabularies.unesco.org/browser/thesaurus/en/page/?uri=http%3A%2F%2Fvocabularies.unesco.org%2Fthesaurus%2Fconcept993

**Comment by jonquet** [comment]
Another example with OpenTheso: https://pactols.frantiq.fr/opentheso/?idc=17695&idt=TH_1 
<img width="479" alt="Capture d’écran 2024-10-16 à 15 27 18" src="https://github.com/user-attachments/assets/3f167616-9234-4f27-82c4-2d4d0c1b5dde">

**Comment by imadbourouche** [comment]
2026 roadmap

**Comment by jonquet** [comment]
Yes this would be good to work on this after the merged with the Project features.
